zbus/src/object_server/interface/mod.rs

```diff
@@ -58,15 +58,15 @@ pub trait Interface: Any + Send + Sync {
         property_name: &str,
         server: &ObjectServer,
         connection: &Connection,
-        header: &Option<message::Header<'_>>,
+        header: Option<&message::Header<'_>>,
```

Comment:
This commit now has 2 seperate logical changes. This one (not covered by the commit log) and the change of `interface` arg. This one is a fix and the other is more an improvement that needs to be applied more consistently. Hence they should be split, assuming we keep the other change.